server/src/search.rs

```diff
@@ -40,6 +41,7 @@ struct SearchFilter {
 #[derive(Debug)]
 struct SearchInput {
     tokens: Vec<SearchToken>,
+    #[allow(dead_code)]
     filter: SearchFilter,
```

Comment:
Ah yes, these two properties are still unused. I already implemented parsing filters in the search query some time ago (eg. `type:room` or `@garching`), but they're not yet used.

"Regular splits" mean that a search token has been split by a space or similar. Unregular splits are e.g. done on `MW2001` -> `MW 2001` because else Meilisearch has issues. I thought that this field might help when filteing/sorting the results from Meilisearch, but this is also not implemented yet.